guides/assets/javascripts/guides.js

```diff
@@ -52,6 +52,25 @@
       });
     }
 
+    // Detecting the mobile state should be done using CSS because that's what
+    // lays out the page, but this can fallback to JavaScript if that is not
+    // available
+    var MOBILE_WIDTH_FROM_STYLE_CSS_BREAKPOINT = 1024
+    var isMobile = window.innerWidth <= MOBILE_WIDTH_FROM_STYLE_CSS_BREAKPOINT;
+
+    if ('matchMedia' in window) {
```

Comment:
matchMedia seems pretty available. I wonder if the else here is necessary

Reply:
A lot of the stuff is available but for some reason the guides are still using `var` among other things. So I've been adding Internet Explorer fallbacks for everything in the past as well.

Unless there comes a consensus guidance on what to support and not support I'm inclined to say that having more support is better than less.

Reply:
This PR has been merged already, but I don't think we need to support Internet Explorer.

Reply:
I agree, but IMO there should be a consensus on what to support (`last 2 versions, not dead, > 0.2%` is common, but not accessible to all of the world and without a build pipeline not great perhaps for *documentation*), and that should be put inside the CONTRIBUTING page for the guides.